src/transactions/dto/transactions.dto.ts

```diff
@@ -0,0 +1,25 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { TransactionType } from "@prisma/client";
+
+export class Transaction {
+    @ApiProperty()
+    transactionId: number;
+
+    @ApiProperty()
```

Comment:
did not need to use @ApiProperty(), anywhere in DTO, it can be handled by swagger configuration and use class-validator to validate the data type of keys